docs/language/informal/enhanced-type-promotion.md

```diff
@@ -125,7 +125,12 @@ The language specifies type promotion out of three pieces:
     closure that is passed to it. To avoid cases like this, the spec defines
     certain variables to be off limits for type promotion.
 
-This proposal keeps the same three general pieces, but refines each of them.
+4.  **Which pairs of types -- declared and promoted -- are allowed for
+    promotion.** The spec says that promotion only applies if the promoted type
+    is a subtype of the declared type.
+
```

Comment:
> The spec says more specific, not subtype.

I think that's the 1.0 spec, but for strong mode / 2.0, I *believe* the intent is to change it to subtype. Is that right, @leafpetersen?